Lib/test/test_cmd_line.py

```diff
@@ -884,6 +884,25 @@ def test_sys_flags_not_set(self):
             PYTHONVERBOSE="1",
         )
 
+    # TODO: RUSTPYTHON
+    @unittest.expectedFailure
+    def test_pythonsafepath_env(self):
+        # Test the PYTHONSAFEPATH environment variable
+        code = "import sys; print(sys.flags.safe_path)"
+        env = dict(os.environ)
+        env.pop('PYTHONSAFEPATH', None)
+        args = (sys.executable, '-P', code)
+
+        proc = subprocess.run(args, stdout=subprocess.PIPE,
+                              universal_newlines=True, env=env)
+        self.assertEqual(proc.stdout.rstrip(), 'False')
+        self.assertEqual(proc.returncode, 0, proc)
+
+        env['PYTHONSAFEPATH'] = '1'
+        proc = subprocess.run(args, stdout=subprocess.PIPE,
+                              universal_newlines=True, env=env)
+        self.assertEqual(proc.stdout.rstrip(), 'True')
+        self.assertEqual(proc.returncode, 0, proc)
```

Comment:
Is this from CPython?

Reply:
No, I just added it. I'll check CPython.

Reply:
This test is not existed in CPython. Is it better deleting this code ?